lucene/core/src/java/org/apache/lucene/geo/GeoEncodingUtils.java

```diff
@@ -127,6 +130,10 @@ public static double decodeLatitude(int encoded) {
   public static double decodeLatitude(byte[] src, int offset) {
     return decodeLatitude(NumericUtils.sortableBytesToInt(src, offset));
   }
+  
+  public static double decodeLatitudeCeil(long encoded) {
```

Comment:
@nknize  could you please review this part?  It's not clear to me why GeoEncodingUtils has dual ceil/floor for encode but not for decode.

Reply:
Or maybe you know @mrkarthik since you felt this addition here was needed?

Reply:
@dsmiley i can add the other method to do the floor.